lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -302,10 +302,10 @@ def write(chunk)
           unless label_map.nil?
             labels = {}
             @label_map.each do |field, label|
-              if record.key?(field)
-                labels[label] = record[field]
-                record.delete(field)
-              end
+              next unless record.key?(field)
+              # label values are required to be strings.
+              labels[label] = record[field].to_s
```

Comment:
to_s is a method on Object so it should always exist.  I added a test case for hash as well, which doesn't seem that useful to me for a user to actually do, but it does work.